src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java

```diff
@@ -0,0 +1,143 @@
+package ca.bc.gov.open.jag.tco.oracledataapi.mapper;
+
+import org.apache.commons.lang3.StringUtils;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.Named;
+
+import ca.bc.gov.open.jag.tco.oracledataapi.model.JJDispute;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.JJDisputeHearingType;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.JJDisputedCount;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.Plea;
+
+@Mapper(componentModel = "spring", injectionStrategy = InjectionStrategy.CONSTRUCTOR) // This is required for tests to work
+public abstract class JJDisputeMapper {
+
+	@Mapping(source = "ticketNumberTxt", target = "ticketNumber")
+	@Mapping(source = "disputeStatusTypeCd", target = "status")
+	@Mapping(source = "disputantSurnameTxt", target = "surname")
+	@Mapping(source = "disputantGiven1Nm", target = "givenNames")
+	@Mapping(source = "offenceLocationTxt", target = "offenceLocation")
+	@Mapping(source = "detachmentLocationTxt", target = "policeDetachment")
+	@Mapping(source = "issuedTs", target = "violationDate")
+	@Mapping(source = "submittedDt", target = "submittedTs")
+	@Mapping(source = "icbcReceivedDt", target = "icbcReceivedDate")
+	@Mapping(source = "disputantBirthDt", target = "contactInformation.birthdate")
+	@Mapping(source = "disputantDrvLicNumberTxt", target = "contactInformation.driversLicenceNumber")
+//	@Mapping(source = "drvLicIssuedProvSeqNo", target = "contactInformation.province") // TODO: field missing in model but exists in database
+//	@Mapping(source = "drvLicIssuedCtryId", target = "drvLicIssuedCtryId")             // TODO: field missing in model but exists in database
+	@Mapping(source = "emailAddressTxt", target = "contactInformation.emailAddress")
+	@Mapping(source = "addressLine1Txt", target = "contactInformation.address")
+	@Mapping(source = "lawyerSurnameNm", target = "lawyerSurname")
+	@Mapping(source = "lawyerGiven1Nm", target = "lawyerGivenName1")
+	@Mapping(source = "lawyerGiven2Nm", target = "lawyerGivenName2")
+	@Mapping(source = "lawyerGiven3Nm", target = "lawyerGivenName3")
+	@Mapping(source = "lawFirmNm", target = "lawFirmName")
+	@Mapping(source = "interpreterLanguageCd", target = "interpreterLanguageCd")
+	@Mapping(source = "courtHearingTypeCd", target = "hearingType", qualifiedByName="mapHearingType")
+	@Mapping(source = "disputeCounts", target = "jjDisputedCounts")
+	@Mapping(source = "entDtm", target = "createdTs")
+	@Mapping(source = "entUserId", target = "createdBy")
+	@Mapping(source = "updDtm", target = "modifiedTs")
+	@Mapping(source = "updUserId", target = "modifiedBy")
+	public abstract JJDispute convert(ca.bc.gov.open.jag.tco.oracledataapi.ords.tco.api.model.JJDispute jjDispute);
```

Comment:
Note: there is a warning in the code here which I left intentionally so these fields will get dealt with some day. These fields are not in ORDS and so have no map to the main application.

`Unmapped target properties: "courthouseLocation, disputantAttendanceType, enforcementOfficer, fineReductionReason, jjAssignedTo, jjDecisionDate, jjDisputeCourtAppearanceRoPs, remarks, timeToPayReason, vtcAssignedTo, vtcAssignedTs, witnessNo".`

Reply:
That's weird. I was able to return remarks as populated for the ticketNumber EA90100004
![remarks-response](https://user-images.githubusercontent.com/98848668/213514921-fb91564b-1047-49c3-8d51-86850d90271c.png)

Reply:
:) It must have been just added. I see the same thing now.  I'll add mapping in the next PR.